code/modules/clothing/suits/armor.dm

```diff
@@ -259,6 +259,12 @@
 	inhand_icon_state = "capspacesuit"
 	body_parts_covered = CHEST|GROIN|ARMS
 
+/obj/item/clothing/suit/armor/vest/capcarapace/jacket
```

Comment:
Modularise

Reply:
helped walk them thru it, should be modular now 👍